### PR TITLE
Bitcoin optech newsletter 129 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2020-12-23-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,956 @@
...
+simplifie le protocole LN et améliore plusieurs aspects de sa sécurité.
+
+<div markdown="1" class="callout" id="optech">
+### Sommaire 2020<br>Bitcoin Optech
+
+Au cours de la troisième année d'existence d'Optech, 10 nouvelles sociétés
+membres se sont jointes à nous<!-- Veriphi, SwanBitcoin, YellowCardIO,
+Xbtogroup, Bitonic, Fidelity Center for Applied Technology, AndgoInc, BTSEcom,
+EdgeWallet, Bitbank_inc -->, nous avons organisé un [atelier taproot][] à
+Londres, publié 51 bulletins d'information hebdomadaires< ! -- 78 à 129 -->,
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2020-12-23-newsletter.md:646]: < ! -- 78 à 129 --> shouldn't render as it should be an HTML comment, can you reformat so this doesn’t render t the screen?

**Review by bitschmidty — CHANGES_REQUESTED** [review]
Can you update the 2021 and 2022 fr end of year newsletters to use this fr translation for 2020?

File: _posts/fr/newsletters/2020-12-23-newsletter.md (hunk 1 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2020-12-23-newsletter.md]: Hello @bitschmidty ,

I make change, but now I have an error 
image

Comment (Copinmalin) [reply _posts/fr/newsletters/2020-12-23-newsletter.md]: OK I think it's OK now,

can you review please @bitschmidty  ?

Comment (Copinmalin) [reply _posts/fr/newsletters/2020-12-23-newsletter.md]: Hello @bitschmidty can you check if it's OK please ?

I would create PR for newsletter but I need to close this one to rebase my repository as discussed

thanks,

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

**Pull request merged**